processor/src/main/resources/org/mapstruct/ap/internal/model/Javadoc.ftl

```diff
@@ -0,0 +1,25 @@
+<#--
+
+    Copyright MapStruct Authors.
+
+    Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<#-- @ftlvariable name="" type="org.mapstruct.ap.internal.model.Javadoc" -->
+/**
+<#list value?split("\n") as line><#nt>*<#if line?has_content> </#if>${line?trim}
+</#list>
+<#if hasAuthors()>
+*
+<#list authors as author> <#nt>* @author ${author?trim}
+</#list>
+</#if>
+<#if hasDeprecated()>
+*
+<#nt>* @deprecated ${deprecated?trim}
+</#if>
+<#if hasSince()>
```

Comment:
We don't need the presence check we can do `<#if since?has_content>`

Reply:
Thank you for all the comments related to the templates: sorry, I am afraid that my FreeMaker skills are a bit rusty.

Reply:
No need to apologise. That's why we have PR, we all learn from each other 😉